packages/compass-connections/src/components/connection-list/connection.tsx

```diff
@@ -300,6 +300,7 @@ function Connection({
         />
         <H3
           className={connectionTitleStyles}
+          as="h3"
```

Comment:
Question, in which cases do we want to set this `as` prop? How is it different from just using  `<H3></H3>`?